frontend/www/js/omegaup/arena/events_socket.ts

```diff
@@ -0,0 +1,256 @@
+import * as time from '../time';
+import * as ui from '../ui';
+import * as api from '../api';
+import {
+  ContestClarificationType,
+  refreshContestClarifications,
+} from './clarifications';
+import clarificationStore from './clarificationsStore';
+import { onRankingChanged, onRankingEvents } from './ranking';
+import { updateRun } from './submissions';
+import { types } from '../api_types';
+
+export enum SocketStatus {
+  Waiting = '↻',
+  Failed = '✗',
+  Connected = '•',
+}
+
+export interface SocketOptions {
+  disableSockets: boolean;
+  problemsetAlias: string;
+  locationProtocol: string;
+  locationHost: string;
+  problemsetId: number;
+  scoreboardToken: null | string;
+  clarificationsOffset: number;
+  clarificationsRowcount: number;
+  currentUsername: string;
+  navbarProblems: types.NavbarProblemsetProblem[];
+  intervalInMiliSeconds: number;
```

Comment:
```suggestion
  intervalInMilliseconds: number;
```

milisegundos es una sóla palabra.

Reply:
Listo.